server/apps/packages/package_service.py

```diff
@@ -146,6 +147,21 @@ def check_package_associations(self, docs):
             for assoc in associations:
                 self.extract_default_association_data(group, assoc)
 
+    def check_not_in_personal_space(self, docs):
```

Comment:
so according to @tictactatic there shouldn't be such limitation, but as a short term fix it's ok